Tests/LinuxMain.swift

```diff
@@ -0,0 +1,6 @@
+import XCTest
+@testable import SnapshotTestingTests
+
+XCTMain([
+    testCase(SnapshotTestingTests.allTests),
```

Comment:
gawd this stuff is gonna get out of sync so fast

Reply:
Why not use sourcery to generate this?  Also you could add a unit test that only runs on macOS that uses the objective-c runtime to make sure every test is listed

Reply:
@kdawgwilk we for sure could do that, just ideally none of that would be necessary and will hopefully be improved soon!